packages/core/package.json

```diff
@@ -11,15 +11,10 @@
   "devDependencies": {
     "@awaitjs/express": "^0.3.0",
     "@ethersproject/units": "^5.0.3",
-    "@nomiclabs/buidler": "^1.3.8",
```

Comment:
these packages are removed from core as they are not part of common and as core has a dependency on common it gets all the required hardhat packages for free.